pkg/apis/eksctl.io/v1alpha4/types.go

```diff
@@ -445,4 +437,16 @@ type (
 		// +optional
 		ALBIngress *bool `json:"albIngress"`
 	}
+
+	// SSHConfig holds all the ssh access configuration to a NodeGroup
+	SSHConfig struct {
```

Comment:
Perhaps we could call it `NodeGroupSSH`, so it is consistent with `NodeGroupIAM` and `NodeGroupSGs`...?

Reply:
I thought about it, then I thought It could represent any SSH key as well. What do you think? I don't have a strong opinion here.

Reply:
I think whatever SSH key it is, it still belongs to a nodegroup :)